### PR TITLE
Feature/4 landing page add button to create a new poll

### LINKED ISSUE #4: Landing Page: Add button to create a new poll
As a user, I'd like to be able to easily create a new poll immediately when accessing the application.

Based on #1 

---

### :thought_balloon: &nbsp; Hints & Thoughts
The button should not have any real functionality right now. We'll create the corresponding page to create a new poll later. First, we'll add all necessary elements to the landing page.

See the [wireframe mockup on Figma](https://www.figma.com/proto/ZrbrdOSBGLSMt1hjgjTvWA/Pollmelo?node-id=2006%3A17&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=2003%3A3)

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [x] should show a new button below the title text "Jetzt abstimmen"
- [x] should be coloured in #E5CB86
- [x] should show a text similar to "Abstimmung erstellen" on top of button (colour: #E95829)

### PR DESCRIPTION
**Pull request by GrafBunsen**
closing #4

### CODE REVIEW

File: assets/css/tailwind.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+    .btn {
+        @apply bg-[#E5CB86] text-[#E95829] p-5 rounded-md text-sm;
```
Comment (martincodes-de) [inline assets/css/tailwind.css]: use our colors like bg-pomelo-red for #e95829 bg-pomelo-yellow for #e5cb86 

in your dev environment you can see [here](http://localhost:3000/_tailwind/) all active colors etc.

File: components/CreatePollButton.vue [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<template>
+  <div>
+    <NuxtLink class="btn" to="/polls/create">Abstimmung erstellen</NuxtLink>
```
Comment (martincodes-de) [inline components/CreatePollButton.vue]: contrast between text and background is too low.

Comment (martincodes-de) [inline components/CreatePollButton.vue]: text is not lightbold / bold, but the mockup shows it as bold

File: layouts/default.vue [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-        <header>
-            <NuxtLink to="/">
-                <img class="m-3" src="~/assets/img/rect24328.png" width="100" height="100" alt="POLLmelo logo" />
-            </NuxtLink>
-            <hr class="h-px my-8 bg-gray-500 border-0 dark:bg-gray-800">
-        </header>    
-        <slot />  
+      <header>
+        <NuxtLink to="/">
+          <img alt="POLLmelo logo" class="m-3" height="100" src="~/assets/img/rect24328.png" width="100"/>
```
Comment (martincodes-de) [inline layouts/default.vue]: please rename logo file name, the name is not speaking

File: pages/index.vue [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
+      <div class="container py-10 mx-auto">
+        <Button/>
+      </div>
     </div>
 </template>
 
 <script setup>
 
-</script>
+import Button from "~/components/CreatePollButton.vue";</script>
```
Comment (martincodes-de) [inline pages/index.vue]: no import needed

File: pages/index.vue [hunk 5]
```diff
@@ -1,16 +1,19 @@
...
-            <h1 class="absolute text-center text-5xl text-pomelo-red font-extrabold top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
-                Jetzt abstimmen!</h1>
-        </div>
+      <div class="relative">
+        <img alt="background-banner" class="object-cover h-60 w-full" src="~/assets/img/banner.png"/>
+        <h1 class="absolute text-center text-5xl text-pomelo-red font-extrabold top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
+          Jetzt abstimmen!</h1>
+      </div>
+      <div class="container py-10 mx-auto">
+        <Button/>
```
Comment (martincodes-de) [inline pages/index.vue]: use name `CreatePollButton`

**Review by martincodes-de — CHANGES_REQUESTED** [review]

**Comment by martincodes-de** [comment]
please add a page `components/button.vue` to show only the button.

insert inside the page script tag
```js
definePageMeta({  layout: ""});
```
to remove the layout

reason: would be great to have a storybook-like environment to develop, test and view components separately

File: components/CreatePollButton.vue (hunk 2 above)
Comment (GrafBunsen) [reply components/CreatePollButton.vue]: the colors for text and background were given in the issue

Comment (GrafBunsen) [reply components/CreatePollButton.vue]: done, also added a shadow

File: layouts/default.vue (hunk 3 above)
Comment (GrafBunsen) [reply layouts/default.vue]: done

File: pages/index.vue (hunk 4 above)
Comment (GrafBunsen) [reply pages/index.vue]: done

**Review by martincodes-de — APPROVED** [review]

**Pull request merged**